libs/cart/src/models/cart-payment.ts

```diff
@@ -1,3 +1,4 @@
 export interface DaffCartPaymentMethod {
-  method: string;
+	method: string;
+	paymentToken?: string;
```

Comment:
Doesn't this need to be an object to accomodate the [extra payment data](https://devdocs.magento.com/guides/v2.3/graphql/payment-methods/authorize-net.html#additional-payment-information)?

Reply:
That's deprecated. We're going to use an authorizenet extension or something

Reply:
Honest answer is though, I don't know

Reply:
ah well I did see the deprecation notice....okay I guess for now we'll just assume that a payment token is sufficient

Reply:
I'll try to figure out more

Reply:
I found (I think) the extension, but it looks like it only supports up to magento 2.3: https://marketplace.magento.com/authorizenet-magento-module-authorizenet.html

I'm talking to Damien about it. Maybe I'll switch to paypal

Reply:
ok ready for review again